tests/realtikvtest/testutils/common.go

```diff
@@ -271,14 +272,20 @@ func createIndexOneCol(ctx *SuiteContext, tableID int, colID int) (err error) {
 	}
 	if err != nil {
 		if ctx.isUnique || ctx.isPK {
-			require.Contains(ctx.t, err.Error(), "Duplicate entry")
+			errorContainsDupKey(ctx.t, err)
 		} else {
 			require.NoError(ctx.t, err)
 		}
 	}
 	return err
 }
 
+func errorContainsDupKey(t *testing.T, err error) {
+	s := err.Error()
+	contains := strings.Contains(s, "Duplicate entry") || strings.Contains(s, "found index conflict records")
```

Comment:
is `found index conflict records` the expected error message when running `add index`?

Reply:
Yes, after this PR, when duplicate key is detected while adding unique key, it reports `Duplicate entry` if disable global sort, and reports `found index conflict records` if enable global sort. We can unify this error message to `found index conflict records` in a future PR

Reply:
lgtm